popup.js

```diff
@@ -169,13 +169,12 @@ document.forms[0].onsubmit = (e) => {
     teddit: getInputValue(data.teddit.el),
     invidious: getInputValue(data.invidious.el),
     scribe: getInputValue(data.scribe.el),
-    bibliogram: getInputValue(data.bibliogram.el),
```

Comment:
Current form value of quetre instance is missing

```
    quetre: getInputValue(data.quetre.el),
```

Reply:
Ah, and another simple miss. I did notice some strange behavior due to this. Thanks again.